src/main/java/org/mitre/synthea/export/BB2RIFExporter.java

```diff
@@ -798,13 +864,76 @@ private void exportPrescription(Person person, long stopTime) throws IOException
   @Override
   public void flush() throws IOException {
     beneficiary.flush();
-    //    beneficiaryHistory.flush();
+    beneficiaryHistory.flush();
     inpatient.flush();
-    //    outpatient.flush();
-    //    carrier.flush();
-    //    prescription.flush();
+    outpatient.flush();
+    carrier.flush();
+    prescription.flush();
+  }
+
+
+  public interface GetEnumValueInterface {
+    public int getEnumValue( String field );
+  }
+
+
+  /**
+   * Exporter Data Builder
+   */
+  public class ExportDataBuilder {
```

Comment:
Seems like this class could be safely merged with ExportConfig so that all of the TSV processing and resulting information is encapsulated in one class?

Reply:
You're right!  I'll do it with ExportConfigEntry as well.